docs/advanced.rst

```diff
@@ -127,6 +127,29 @@ referencing the ``changed_by`` field:
 
 Admin integration requires that you use a ``_history_user.setter`` attribute with your custom ``_history_user`` property (see :ref:`admin_integration`).
 
+Custom ``history_id``
+---------------------
+
+If you want to use something other then an ``AutoField`` for the history
+table primary key you can pass in a ``history_id_field``.  A common use case
+for this would be to use a ``UUIDField``.
+
+For any field provided it will automatically set ``primary_key=True`` and
+``editable=False``.
+
+.. code-block:: python
+
+    import uuid
+    from django.db import models
+    from simple_history.models import HistoricalRecords
+
+    class Poll(models.Model):
+        question = models.CharField(max_length=200)
+        pub_date = models.DateTimeField('date published')
+        history = HistoricalRecords(
```

Comment:
So this only works if I add history to the model using `HistoricalRecords`? You're also going to want to edit the `register` function since that is the other way to give a model history

Reply:
It does work with `register`, I will update the docs and unit tests so that is tested.

Reply:
Oh yes, it gets passed through `**records_config`. Please update docs and tests for that